src/main/resources/application.yml

```diff
@@ -132,10 +132,10 @@ labresults:
       3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
       -----END RSA PRIVATE KEY-----}
     subCAcert: ${LAB_RESULTS_MESH_SUB_CA:}
-    recipientToMailboxIdMappings: ${LAB_RESULTS_MESH_RECIPIENT_MAILBOX_ID_MAPPINGS:000000004400001=lab_results_mailbox
-      000000024600002=lab_results_mailbox
-      000000183500005=lab_results_mailbox
-      110000044300001=lab_results_mailbox}
+    recipientToMailboxIdMappings: ${LAB_RESULTS_MESH_RECIPIENT_MAILBOX_ID_MAPPINGS:000000004400001=lab_results_ack_mailbox
```

Comment:
The test mailbox name from docker-compose (`lab_results_ack_mailbox`) will be passed here when testing locally, so shouldn't we just keep the default mailbox name still as `lab_results_mailbox` here?
Like we do for `wakeupIntervalInMilliseconds` which is "60000" by default

Reply:
In real life "our" inbox will in fact be different than the one we send nhsacks to (owned by the lab).
Let's leave the other defaults in the app as they are as there is a chance the test partners don't alter that when deploying to prod. This will be a breaking change for them. On the other hand `recipientToMailboxIdMappings` is something that they have to change in order to make prod to work.